public/img/SixteenSegDisplay.svg

```diff
@@ -0,0 +1 @@
7l-6-6z"/><path d="M161,56h71v1l-7,7l-8,8h-1v2l-20,20h-1v2l-19,19h-1v2l-13,13h-1z"/><path d="M79,60h1l11,11c0,2,1,2,16,17c0,2,1,2,16,17c0,2,1,2,16,17c0,2,1,2,8,9l2,2v1h-70z"/><path d="M234,60h1v74h-71v-1l5-5l8-8h1v-2l20-20h1v-2l19-19h1v-2z"/><path d="M79,143h70v1l-5,5l-1,1h-1v2l-13,13h-1v2l-12,12h-1v2l-12,12h-1v2l-13,13h-1v2l-8,8h-1z"/><path d="M164,143h71v75h-1l-11-11c0-2-1-2-16-17c0-2-1-2-16-17c0-2-1-2-17-18c0-2-1-2-8-9l-2-2z"/><path d="M151,147h1v75h-70v-1l5-5l3-3h1v-2l13-13h1v-2l12-12h1v-2l12-12h1v-2l13-13h1v-2z"/><path d="M161,147h1l9,9c0,2,1,2,16,17c0,2,1,2,16,17c0,2,1,2,17,18c0,2,1,2,6,7l6,6v1h-71z"/></g><g style="fill:rgb(211,212,211);fill-rule:evenodd;stroke:rgb(211,212,211);stroke-width:0.1;" id="l2"><path d="M78,47h158c0,5,1,5,3,7v2h5v79h-8v8h8v79c-5,0-5,2-7,4h-1v5h-158c0-5-1-5-3-7v-2h-5v-79h8v-8h-8v-79c5,0,5-2,7-4h1zm153,11v-1l1-1h-71v74l13-13h1v-2l9-9l10-10h1v-2l20-20h1v-2l14-14zm-152,85v75l8-8h1v-2l13-13h1v-2l12-12h1v-2l12-12h1v-2l13-13h1v-2l7-7zm120-47v2l-20,20h-1v2l-14,14h71v-74l-15,15h-1v2l-7,7l-12,12zm-48,34h1v-74h-70l12,12v2h1l15,15v2h1l15,15v2h1l15,15v2h1l5,5l3,3zm-33,52v2l-12,12h-1v2l-13,13h-1v2l-9,9h70v-75l-6,6h-1v2l-13,13h-1v2l-12,12zm-3-85h-1l-7-7v-2h-1l-5-5l-10-10v-2h-1l-11-11v74h70l-10-10v-2h-1l-15-15v-2h-1l-7-7zm46,50v75h71l-12-12v-2h-1l-16-16v-2h-1l-15-15v-2h-1l-15-15v-2h-1zm37,33h1l8,8v2h1l5,5l10,10v2h1l11,11v-75h-71l10,10v2h1l16,16v2h1l5,5l1,1z"/><path d="M251,233h10v10h-10z"/></g><g style="fill:rgb(4,5,4);fill-rule:evenodd;stroke:rgb(4,5,4);stroke-width:0.1;" id="l3"><path d="M38,20h21v7h-14v37h-7zm33,0h34v7h-34z"/><path d="M117,20h34v7h-34zm46,0h34v7h-34zm46,0h34v7h-34z"/><path d="M255,20h20v44h-7v-37h-13zm-217,56h7v57h-7z"/><path d="M268,76h7v57h-7zm-230,69h7v57h-7z"/><path d="M268,145h7v57h-7zm-230,69h7v36h14v7h-21z"/><path d="M268,214h7v34h-7zm-197,36h34v7h-34zm46,0h34v7h-34z"/><path d="M163,250h34v7h-34zm46,0h34v7h-34zm46,0h11v7h-11z"/></g><g style="fill:rgb(6,149,24);fill-rule:evenodd;stroke:rgb(6,149,24);stroke-width:0.1;" id="l4"><path d="M62,17h6l2,1l1,1l1,3v6l-3,3h-8l-3-3v-6l1-3l1-1zm46,0h6l2,1l1,1l1,3v6l-3,3h-8l-3-3v-6l1-3l1-1zm46,0h6l2,1l1,1l1,3v6l-3,3h-8l-3-3v-6l1-3l1-1zm46,0h6l2,1l1,1l1,3v6l-3,3h-8l-3-3v-6l1-3l1-1z"/><path d="M246,17h6l2,1l1,1l1,3v6l-3,3h-8l-3-3v-6l1-3l1-1zm-206,46h6l3,3v8l-3,3h-6l-3-1l-1-1l-1-2v-6l1-2l1-1zm230,0h4l3,1l1,1l1,2v6l-1,2l-1,1l-3,1h-4l-3-1l-2-2v-8l2-2zm-230,69h6l3,3v8l-3,3h-6l-3-1l-1-1l-1-2v-6l1-2l1-1z"/><path d="M270,132h4l3,1l1,1l1,2v6l-1,2l-1,1l-3,1h-4l-3-1l-2-2v-8l2-2zm-230,69h6l3,3v8l-3,3h-6l-3-1l-1-1l-1-2v-6l1-2l1-1zm230,0h4l3,1l1,1l1,2v6l-1,2l-1,1l-3,1h-4l-3-1l-2-2v-8l2-2zm-209,46h8l2,2l1,3v4l-1,3l-1,1l-2,1h-6l-2-1l-1-1l-1-3v-4l1-3z"/><path d="M107,247h8l2,2l1,3v4l-1,3l-1,1l-2,1h-6l-2-1l-1-1l-1-3v-4l1-3zm46,0h8l2,2l1,3v4l-1,3l-1,1l-2,1h-6l-2-1l-1-1l-1-3v-4l1-3zm46,0h8l2,2l1,3v4l-1,3l-1,1l-2,1h-6l-2-1l-1-1l-1-3v-4l1-3zm46,0h8l2,2l1,3v4l-1,3l-1,1l-2,1h-6l-2-1l-1-1l-1-3v-4l1-3z"/><path d="M270,247h4l3,1l1,1l1,2v6l-1,2l-1,1l-2,1h-6l-2-1l-1-1l-1-3v-4l1-3l1-1z"/></g></svg>
```

Comment:
Remember to update the svg icon to reflect the new layout.
If appropriate, feel free to use rect and circle; they make the svg smaller and more understandable as well.

Reply:
Sure! I'll do that.